modules/oe_webtools_cookie_consent/test/src/Kernel/FilterPluginKernelTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools_cookie_consent\Kernel;
+
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\filter\FilterPluginCollection;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\language\Entity\ConfigurableLanguage;
+
+/**
+ * Tests filter plugins.
+ */
+class FilterPluginKernelTest extends KernelTestBase {
```

Comment:
Do we have any additional reasons why we have such an agnostic class name? Do we plan in future within the oe_webtools_cookie_consent module test other filters?